### ISSUE TITLE
Multiple skip_rows conditions that apply to the same row causes error

### ISSUE THREAD

**Issue by kyao** [usc-isi-i2/t2wml#161]
**Describe the bug**

![image](https://user-images.githubusercontent.com/1087657/90196245-313f3600-dd80-11ea-8359-c65c6177787e.png)

```
statementMapping:
  region:
  - bottom: 84  # 431
    left: C
    right: L  # AJ
    top: 16
    skip_rows:
      - =value[B,$row] == ""
      - =value[D,$row] == ""
  template:
    item: =item[B, $row, "main subject"]
    property: =item[$col-$n, 12, "property"]
    qualifier:
    - calendar: Q1985727
      format: '%Y'
      precision: year
      property: P585
      time_zone: 0
      value: =regex(value[$col,15], "(....).*/")
    - property: P2006020004
      value: QEth-crops
    value: =value[$col, $row]
```

I will send a tar of the relevant files to reproduce the error.

**Comment by szeke** [comment]
You can work around by using the two value statements in the same line using `or`

**Comment by kyao** [comment]
For `or` this works:
```
=(value[B,$row] == "" or value[D,$row] == "")
```
But not the following, which fails with the same error
```
=value[B,$row] == "" or value[D,$row] == ""
```

**Comment by devowit** [comment]
fixed in usc-isi-i2/t2wml-api@bc775e1ac

**Comment by devowit** [comment]
version 2.1.1
